plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryClient.java

```diff
@@ -13,6 +13,7 @@
  */
 package io.trino.plugin.bigquery;
 
```

Comment:
Please update commit message to `Move DestinationTableBuilder to BigQueryClient`

Reply:
Done.